tools/dockerfile/grpc_iwyu/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Copyright 2021 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM debian:bullseye
+
+RUN apt-get update && apt-get install -y clang-11 llvm-11-dev libclang-11-dev clang-format-11 jq git cmake python
+
```

Comment:
Maybe we can define CLANG_FORMAT= like https://github.com/grpc/grpc/blob/86e0634fa415dd5d59dda03a847b4a7896c543e3/templates/tools/dockerfile/grpc_clang_format/Dockerfile.template#L21 so that iwyu,sh script doesn't need to specify `clang-format-11`

Reply:
Done